src/ode_solve.jl

```diff
@@ -23,7 +23,14 @@ of the physics-informed neural network which is used as a solver for a standard
   which thus uses the random initialization provided by the neural network library.
 
 ## Keyword Arguments
-
+* `additional_loss`: A function additional_loss(phi, θ) where phi are the neural network trial solutions,
+                     θ are the weights of the neural network(s).
+example: 
+    ts=[t for t in 1:100]
+    (u_, t_) = (analytical_func(ts), ts)
+    function additional_loss(phi, θ)
+        return sum(sum(abs2, [phi(t, θ) for t in t_] .- u_)) / length(u_)
+    end
```

Comment:
This isn't in the right spot. Make an example section.

Reply:
under this argument's description? or along with the lower example?